hail/Makefile

```diff
@@ -77,17 +77,17 @@ DEV_SALT := $(shell whoami)-dev
 endif
 
 CLOUD_LOC = gs://hail-common/hailctl/dataproc/$(DEV_SALT)/$(HAIL_PIP_VERSION)/$(SHORT_REVISION)/$(FILENAME)
-RESOURCES_TO_COPY = $(shell ls ../hailctl/python/hailctl/dataproc/resources)
-../hailctl/python/hailctl/deploy.yaml:
-	rm -f ../hailctl/python/hailctl/deploy.yaml
-	echo "dataproc:" >> ../hailctl/python/hailctl/deploy.yaml
+RESOURCES_TO_COPY = $(shell ls python/hailctl/dataproc/resources)
+python/hailctl/deploy.yaml:
+	rm -f python/hailctl/deploy.yaml
```

Comment:
Do you need this line? Isn't this what .PHONY does?

Reply:
not really. But I didn't change this line in the PR -- just the paths these things point to, so I think the change is out of scope here

Reply:
I think in a previous version I was appending to the file in various places, and wanted to be sure I was starting fresh